controllers/fractie.ts

```diff
@@ -12,7 +12,9 @@ async function forBestuursperiode(
   bestuursperiodeId: string,
   onafhankelijk,
 ): Promise<Array<string>> {
-  const isBestuursperiode = await bestuursperiode.isValidId(bestuursperiodeId);
+  const isBestuursperiode = await areIdsValid(RDF_TYPE.BESTUURSPERIODE, [
```

Comment:
Also used te areIdsValid here because than we could remove the bestuursperiode.ts file in data-access